src/FormElementManagerFactory.php

```diff
@@ -11,31 +11,22 @@
 use Interop\Container\ContainerInterface;
 use Laminas\ServiceManager\AbstractPluginManager;
 use Laminas\ServiceManager\Config;
-use Laminas\ServiceManager\FactoryInterface;
+use Laminas\ServiceManager\Factory\FactoryInterface;
 use Laminas\ServiceManager\ServiceLocatorInterface;
 
 use function is_array;
 use function method_exists;
 
 class FormElementManagerFactory implements FactoryInterface
```

Comment:
This class is not final and thus extendable by endusers.

We should keep it as is even tho most of it is not used anymore. Marking methods as deprecated aswell as the `creationOptions` property would be the best way.

Reply:
Right: I've `@deprecated` attribute and method, and still removed what can be safely removed